components/autofill/android/src/test/java/mozilla/appservices/autofill/AutofillTest.kt

```diff
@@ -1,37 +1,33 @@
 import mozilla.appservices.Megazord
 import mozilla.appservices.autofill.Store
 import mozilla.appservices.syncmanager.SyncManager
-import org.junit.Assert
-import org.junit.Rule
-import org.junit.Test
-import org.junit.rules.TemporaryFolder
-import org.junit.runner.RunWith
-import org.robolectric.RobolectricTestRunner
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.io.TempDir
 import org.robolectric.annotation.Config
+import java.io.File
 
-@RunWith(RobolectricTestRunner::class)
 @Config(manifest = Config.NONE)
 class AutofillTest {
-    @Rule
-    @JvmField
```

Comment:
Stackoverflow says that we need to keep the `@JvmField`: https://stackoverflow.com/questions/68202376/how-to-use-junit-5s-tempdir-with-kotlin